app/build.gradle

```diff
@@ -19,10 +19,12 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'androidx.appcompat:appcompat:1.0.0'
```

Comment:
Is this the latest?  I thought it was 1.0.2.

Reply:
that latest is `1.1.0` right now, but the [Migration guide](https://developer.android.com/jetpack/androidx/migrate) still points to `1.0.0`. Different dependencies have different versions now, so I use migration guide as source of truth for all of them.